Doc/c-api/module.rst

```diff
@@ -77,8 +78,9 @@ Module Objects
       single: __name__ (module attribute)
       single: SystemError (built-in exception)
 
-   Return *module*'s :attr:`__name__` value.  If the module does not provide one,
-   or if it is not a string, :exc:`SystemError` is raised and ``NULL`` is returned.
+   Return *module*'s :attr:`~module.__name__` value.  If the module does not
+   provide one, or if it is not a string, :exc:`SystemError` is raised and
```

Comment:
I notice below that we say `module.__file__` must be a "unicode string" but here we say `module.__name__` must be a "string".  These should probably be made consistent.

Reply:
Should probably just say "string"; "unicode string" feels like a Python 2-ism.

Reply:
Yup, I'll change them all to "string"s. All strings these days are unicode 😄